### DISCUSSION TITLE
xorgxrdp v0.9.19

### DISCUSSION THREAD

**Discussion by metalefty** [neutrinolabs/xorgxrdp#233] [Announcements]
## General announcements

You might be surprised that xorgxrdp version is suddenly jumped from `v0.2.18` to `v0.9.19`. We decided to bump up version from v0.2 to v0.9 to indicate xorgxrdp v0.9 is compatible with xrdp v0.9. 

We also made a tag v0.2.19.  Both v0.2.19 and v0.9.19 are minor version up from v0.2.18 and identical.  For upcoming xorgxrdp v1.0 is not compatible with xrdp v0.9  and is for upcoming xrdp v1.0.

### Too fast scroll mitigation

Workaround for too fast vertical scrolling on touchpad (#150) has been implemented. Define an environment variable `XRDP_XORG_TOUCHPAD_SCROLL_HACK` and set to yes, true, on or 1 to mitigate too fast scroll. More specifically, edit xrdp's `sesman.ini` as following and restart xrdp-sesman. 

```
[SessionVariables]
(some existing lines)
XRDP_XORG_TOUCHPAD_SCROLL_HACK=yes
```

This is a temporary workaround and may be removed in future version.

## What's Changed
* Fix build break in GitHub workflow by @Nexarian in https://github.com/neutrinolabs/xorgxrdp/pull/213
* populate DRI2InfoRec driverNames by @akarl10 in https://github.com/neutrinolabs/xorgxrdp/pull/216
* Update for unification of monitor processing by @Nexarian in https://github.com/neutrinolabs/xorgxrdp/pull/212
* xrdpdri2.c typo by @akarl10 in https://github.com/neutrinolabs/xorgxrdp/pull/218
* Add glamor to CI by @matt335672 in https://github.com/neutrinolabs/xorgxrdp/pull/219
* use damage to track any lost screen changes by @jsorg71 in https://github.com/neutrinolabs/xorgxrdp/pull/186
* Revert "Update for unification of monitor processing" by @metalefty in https://github.com/neutrinolabs/xorgxrdp/pull/221
* [v0.9] mouse: workaround for too fast vertical scroll by @metalefty in https://github.com/neutrinolabs/xorgxrdp/pull/229
* [v0.9] Merge #228 for v0.9 by @metalefty in https://github.com/neutrinolabs/xorgxrdp/pull/230
* [v0.9] Ignore screen size changes which don't change anything (#203) by @metalefty in https://github.com/neutrinolabs/xorgxrdp/pull/232
* Bump version to v0.9.19 by @metalefty in https://github.com/neutrinolabs/xorgxrdp/pull/231

## New Contributors
* @akarl10 made their first contribution in https://github.com/neutrinolabs/xorgxrdp/pull/216

**Full Changelog**: https://github.com/neutrinolabs/xorgxrdp/compare/v0.2.18...v0.9.19

<hr /><em>This discussion was created from the release <a href='https://github.com/neutrinolabs/xorgxrdp/releases/tag/v0.9.19'>xorgxrdp v0.9.19</a>.</em>

**Comment by bsmojver** [comment]
I think the signature files were missed. Would you mind uploading those?

**Reply by metalefty** [reply]
@bsmojver Uploaded.
